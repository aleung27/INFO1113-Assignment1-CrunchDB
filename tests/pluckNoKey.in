PLUCK t 1
SET t 0 29 3
PLUCK t 1
PLUCK t 1
PLUCK t 1
PLUCK t 1
LIST ENTRIES
