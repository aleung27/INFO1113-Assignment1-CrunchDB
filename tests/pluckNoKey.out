> no such key

> ok

> 0

> 29

> 3

> index out of range

> t []

